CodeEdit/Features/AppPreferences/Model/Text Editing/TextEditingPreferences.swift

```diff
@@ -97,7 +97,7 @@ extension AppPreferences {
         /// Indicates whether or not to use a custom font
         var customFont: Bool = false
 
-        /// The font size for the custom font
+        /// The font size for the font
```

Comment:
Why take the word "custom" out? It is relevant in this context. Please put it back in. It is difficult to merge this when you keep making unrelated changes. It is like trying to hit a moving target.